projects/admin/src/app/service/record-permission.service.ts

```diff
@@ -54,6 +56,16 @@ export class RecordPermissionService {
     return this._httpClient.get<RecordPermission>(url, this._httpOptions);
   }
 
+
+  /**
+   * Get roles that the current user can manage
+   * @return an observable on allowed roles management
+   */
+  getRolesManagementPermissions(): Observable<any> {
+    const url = 'api/patrons/roles_management_permissions';
+    return this._httpClient.get(url, this._httpOptions);
```

Comment:
As `url` is used only one time, I propose to put the string directly in `get` method.

Reply:
done